packages/expect/src/__tests__/__snapshots__/toThrowMatchers.test.js.snap

```diff
@@ -166,92 +166,88 @@ Received has value: <red>\\"a string\\"</>"
 `;
 
 exports[`.toThrowError() invalid arguments 1`] = `
-"<dim>expect(</><red>received</><dim>)[.not].toThrowError(</><green>expected</><dim>)</>
+"<dim>expect(</><red>received</><dim>).</>not<dim>.</>toThrowError<dim>(</><green>expected</><dim>)</>
 
-<bold>Matcher error</>: <green>expected</> value must be a string or regular expression or Error
+<bold>Matcher error</>: <green>expected</> value must be a string or regular expression or class or error
 
 Expected has type:  number
 Expected has value: <green>111</>"
 `;
 
 exports[`.toThrowError() promise/async throws if Error-like object is returned did not throw at all 1`] = `
-[Error: [2mexpect([22m[31mfunction[39m[2m).toThrow([22m[32mundefined[39m[2m)[22m
+[Error: [2mexpect([22m[31mreceived[39m[2m).[22mrejects[2m.[22mtoThrowError[2m()[22m
```

Comment:
Can you add the missing ansi code to `pretty-format`? or is it something else?

(I'm aware it was not properly serialized earlier either)

Reply:
Yeah, I will look in the morning why promise/async snapshots are different from others.

Reply:
Aha, wherever else this might happen in async tests, here is the solution:

```js
expect(err && err.message).toMatchSnapshot();
```